integration_tests/tfjs2keras/tfjs_save.ts

```diff
@@ -0,0 +1,142 @@
+/**
+ * @license
+ * Copyright 2018 Google LLC
+ *
+ * Use of this source code is governed by an MIT-style
+ * license that can be found in the LICENSE file or at
+ * https://opensource.org/licenses/MIT.
+ * =============================================================================
+ */
+
+import * as tfl from '@tensorflow/tfjs-layers';
+import * as fs from 'fs';
+import {join} from 'path';
+
+// Multi-layer perceptron (MLP).
+function exportMLPModel(exportPath: string): void {
+  const model = tfl.sequential();
+  // Test both activations encapsulated in other layers and as standalone
+  // layers.
+  model.add(
+      tfl.layers.dense({units: 100, inputShape: [200], activation: 'relu'}));
+  model.add(tfl.layers.dense({units: 50, activation: 'elu'}));
+  model.add(tfl.layers.dense({units: 24}));
+  model.add(tfl.layers.activation({activation: 'elu'}));
+  model.add(tfl.layers.dense({units: 8, activation: 'softmax'}));
+  fs.writeFileSync(exportPath, model.toJSON());
+}
+
+// Convolutional neural network (CNN).
+function exportCNNModel(exportPath: string): void {
```

Comment:
Going to approve the PR as it is, but how hard would it be to evolve the system to use either directories in tfjs-examples or tfjs-models as the source of models rather than hard-coding them here.